docs/Contributing/guides/cli/README.md

```diff
@@ -0,0 +1,13 @@
+# Fleet CLI Development Guides
+
+This directory contains guides for CLI development in Fleet.
+
+## CLI Development
+
+- [Fleetctl Apply](cli/fleetctl-apply.md) - Guide for using fleetctl apply
+
+## Related Resources
+
+- [API for Contributors](../../workflows/api-for-contributors.md) - Workflow for API development
```

Comment:
Similar "this title and probably file location feels wrong" comment to with the API README.

Reply:
Same bad link. Thanks.